notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>I'm not sure that we can mention explicitly My Little Pony in notebook due to trademarks, can this info be committed?</p><p>Better to provide info about how to change to other models in this comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Changed